css/css-ui-3/text-overflow-017.html

```diff
@@ -0,0 +1,37 @@
+﻿<!DOCTYPE html>
+<html class="reftest-wait">
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: interacting with the ellipsis</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#text-overflow">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem interact dom">
+<meta name="assert" content="Ellipsing must not affect dispatching of pointer events.">
```

Comment:
I'd say it probably should go to the span, but I agree the spec is vague enough that this is probably undefined for now. Would you mind if this was filed as an issue against the text level as something to clarify, and if we kept the tests of this level at what I've written in this test?